onnxruntime/test/python/quantization/test_qdq.py

```diff
@@ -208,23 +207,23 @@ def test_qdq_extra_options_2(self):
         # QDQ pair shoud not be added to node's output
         for node in quantizer.model.nodes():
             if node.name == "MatMul1":
-                self.assertTrue("T_DequantizeLinear_1" in node.input)
+                self.assertTrue("T_DequantizeLinear_Output_1" in node.input)
```

Comment:
Prefer https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertNotIn for clarity when there is an error